src/morphology_workflows/repair.py

```diff
@@ -434,3 +464,7 @@ def make_release(df, _, zero_diameter_path, unravel_path, repair_path, extension
             df.loc[df["is_valid"], f"repair_morph_db_path_{extension[1:]}"] = (
                 _repair_path / "neurondb.xml"
             )
+
+    # we drop duplicate to preserve df size
+    df.drop(index=df.index[n_rows:], axis=0, inplace=True)
+    df.set_index("morph_name", inplace=True)
```

Comment:
Why not creating a temporary DF with duplicated rows instead of adding then removing rows in the external DF? If an exception is raised when the DF has duplicated rows it might break DVF.

Reply:
so we get the data added in new columns, if I don't do things in place, it does not write the new data in the .csv. This is weird, I mentioned above.

Reply:
This is the way DVF works. A `SetValidationTask` should not return anything (the `ValidationResultSet` is just an internal class of DVF but I don't we have any reason to use it, though the name is a bit confusing indeed), it should only update the cells of the input DF (and thus it should not add or delete any row).
But it's still possible to build duplicates without adding rows in the DF. I showed he general idea in #61 .